components/grouped-actions.js

```diff
@@ -68,56 +69,43 @@ function GroupedActions({idVoie, numeros, selectedNumerosIds, resetSelectedNumer
   const handleConfirm = useCallback(async event => {
     event.preventDefault()
 
-    const data = {selectedVoieId, selectedToponymeId, numeros: selectedNumeros, positionType}
-    const body = data.numeros
-    const type = data.positionType
-
-    setIsLoading(true)
-
-    const commentCondition = r => {
+    const commentCondition = c => {
       if (removeAllComments) {
         return null
       }
 
-      if (comment === '') {
-        return r.comment || null
-      }
-
-      if (r.comment) {
-        return `${r.comment} , ${comment}`
+      if (c) {
+        return c
       }
-
-      return comment
     }
 
-    body.map(r => {
-      r.voie = selectedVoieId
-      r.toponyme = selectedToponymeId === '' ? null : selectedToponymeId || r.toponyme
-      r.positions.forEach(position => {
-        if (type) {
-          position.type = type
-        }
-      })
-
-      r.comment = commentCondition(r)
-
-      if (certifie !== null) {
-        r.certifie = certifie
+    const getCertifie = c => {
```

Comment:
Idem ici pour le nom de variable. Il est même préférable de préfixer le nom par `is` car il me semble que c'est un booléen.